charts/reform-scan-blob-router/values.preview.template.yaml

```diff
@@ -9,18 +9,36 @@ java:
     STORAGE_ACCOUNT_SECONDARY_KEY:
       secretRef: storage-secret-{{ .Release.Name }}
       key: accessKey
+    STORAGE_BULKSCAN_ACCOUNT_NAME:
+      secretRef: storage-secret-cft-{{ .Release.Name }}
+      key: storageAccountName
+    STORAGE_BULKSCAN_ACCOUNT_KEY:
+      secretRef: storage-secret-cft-{{ .Release.Name }}
+      key: accessKey
   environment:
+    SPRING_CLOUD_PROPERTIESVOLUME_ENABLED: false
```

Comment:
why do we need this change ?

Reply:
otherwise it bootstraps  from secret and overrides osba connections

Reply:
as discussed please add a comment here as this needs to be removed once we have key vault config

Reply:
added